_posts/2018-12-05-graphql-union-vs-interface.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
Can we update the filename to match the new date, too?

Reply:
good point! done!